src/api/libcellml/analyservariable.h

```diff
@@ -121,13 +120,33 @@ class LIBCELLML_EXPORT AnalyserVariable
     VariablePtr variable() const;
 
     /**
-     * @brief Get the @ref AnalyserEquation for this @ref AnalyserVariable.
+     * @brief Get the number of equations used to compute this @ref AnalyserVariable.
      *
-     * Return the @ref AnalyserEquation for this @ref AnalyserVariable.
+     * Return the number of equations used to compute this @ref AnalyserVariable.
      *
-     * @return The @ref AnalyserEquation.
+     * @return The number of equations used to compute this @ref AnalyserVariable.
      */
-    AnalyserEquationPtr equation() const;
+    size_t equationCount() const;
+
+    /**
+     * @brief Get the equations used to compute this @ref AnalyserVariable.
+     *
+     * Return the equations used to compute this @ref AnalyserVariable.
+     *
+     * @return The equations.
```

Comment:
This is not what is returned, indeed not helpful for a reader of the API.

Reply:
agreed.

Reply:
d3ea8b9f